src/arrayMethodSort.js

```diff
@@ -4,8 +4,28 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (compareFunction) {
     // write code here
+    const copyArr = [...this];
+    let defaultFn = compareFunction;
+
+    if (!compareFunction) {
+      defaultFn = (a, b) => String(a) > String(b);
+    }
+
+    for (let i = 0; i < copyArr.length - 1; i++) {
+      for (let j = 0; j < copyArr.length - 1 - i; j++) {
+        if (defaultFn(copyArr[j], copyArr[j + 1]) > 0) {
```

Comment:
The comparison function should return a number less than 0, exactly 0, or greater than 0. Using `> 0` is correct, but ensure to handle all three cases as per the specification of the sort method.